docs/website/versioned_docs/version-3.0.0/user-guides/quickstart.md

```diff
@@ -690,6 +861,411 @@ Your Devfile has been successfully deployed
 
 Your application has now been deployed to the Kubernetes cluster with Deployment, Service, and Ingress resources.
 
+Test your application by visiting the `DOMAIN_NAME` variable that you had set in the `devfile.yaml`.
+
+  </TabItem>
+  <TabItem value="java" label="Java (Spring Boot)">
+
+#### 1. Containerize the application
+
+In order to deploy our application, we must containerize it in order to build and push to a registry. Create the following `Dockerfile` in the same directory:
+
+```dockerfile
+FROM registry.access.redhat.com/ubi8/openjdk-11 as builder
+
+USER jboss
+WORKDIR /tmp/src
+COPY --chown=jboss . /tmp/src
+RUN mvn package
+
+FROM registry.access.redhat.com/ubi8/openjdk-11
+COPY --from=builder /tmp/src/target/*.jar /deployments/app.jar
+```
+
+#### 2. Modify the Devfile
+
+Let's modify the `devfile.yaml` and add the respective deployment code.
+
+`odo deploy` uses Devfile schema **2.2.0**. Change the schema to reflect the change:
+
+```yaml
+# Deploy "kind" ID's use schema 2.2.0+
+schemaVersion: 2.2.0
+```
+
+Add the `variables` section:
+
+```yaml
+# Add the following variables code anywhere in devfile.yaml
+# This MUST be a container registry you are able to access
+variables:
+  CONTAINER_IMAGE: quay.io/MYUSERNAME/java-springboot-odo-example
```

Comment:
Can you add a note asking the user to make sure they change this? I made a mistake of not looking at the comment and tried `odo deploy`.

Reply:
Sure! I'll add a note.